src/auth/service/authenticator/authenticator.go

```diff
@@ -78,10 +77,11 @@ func (authen *Authenticator) AuthenticateAPIKey(_ context.Context, apiKey []byte
 		return nil, fmt.Errorf("failed to hash api key")
 	}
 
-	claims, ok := authen.apiKeyClaims[base64.StdEncoding.EncodeToString(authen.hasher.Sum(nil))]
+	apiKeySha256 := fmt.Sprintf("%x", authen.hasher.Sum(nil))
```

Comment:
why do you need `fmt.Sprintf` here? Seems that you get back []byte and you can simply do `string(authen.hasher.Sum(nil))`

Reply:
resolved